db/influx.go

```diff
@@ -184,6 +249,19 @@ func updateRelayCount(current api.RelayCounts, relayResult string, count int64)
 	return current, nil
 }
 
+func updateRelayCountOrigin(current api.RelayCounts, relayResult string, count int64) (api.RelayCounts, error) {
+	switch {
+	case relayResult == "_result":
+		current.Success += int64(count)
+	//TODO: Update the condition when relays fails after we define it
```

Comment:
`// TODO: Update the condition when relays fails after we define it`

Reply:
Right now the bucket will always return the count and the default result string. To keep the same format as the front is expecting I leaved the response as `RelayCounts` but we're unable to identify if it's those X relays were succeeded or failed